test/ueberauth_linkedin_test.exs

```diff
@@ -1,8 +1,5 @@
 defmodule UeberauthLinkedinTest do
   use ExUnit.Case
-  doctest UeberauthLinkedin
 
-  test "the truth" do
```

Comment:
How will we know, then??

Reply:
We just have to trust the code... Like how we trust the rest of the untested code XD